src/app.constants.ts

```diff
@@ -69,6 +69,7 @@ export const PATH_NAMES = {
   CHANGE_SECURITY_CODES_CONFIRMATION: "/change-codes-confirmed",
   PASSWORD_RESET_REQUIRED: "/password-reset-required",
   UNAVAILABLE_PERMANENT: "/unavailable-permanent",
+  UNAVAILABLE: "/unavailable",
```

Comment:
A check here - should this be `unavailable-temporary`? I see the ticket is confused on this (says both `unavailable` and `unavailable-temporary` but the latter seems to follow the pattern of the above

Reply:
Ahh yes. Thanks for the reminder!